documents/README.md

```diff
@@ -80,25 +83,25 @@ Literate code blocks support the following attributes:
 - `s` (or `siding`) is a shorthand for `isolated follows=default`.
   This attribute does not expect a value.
 
-- `interactive` (or `interactive=true`) makes visualised specs interactive if _grammar of interaction_ is used within the specs.
+- `interactive` (or `interactive=true`) makes specs interactive if they include interactive elements such as [elm-vegalite selections](https://package.elm-lang.org/packages/gicentre/elm-vegalite/latest/VegaLite#selection) and [elm-vega event handling](https://package.elm-lang.org/packages/gicentre/elm-vega/latest/Vega#evHandler).
 
-The order of `l`, `v`, `r` or `j` in attributes determines the order of rendering.
+The order of `l`, `v`, `r`, `m` or `j` in attributes determines the order of rendering.
 
-Defining `l` to make the code block literate is not necessary if `v`, `r` or `j` are already given (this implies `l=hidden`).
+Defining `l` to make the code block literate is not necessary if `v`, `r`, `m` or `j` are already given (this implies `l=hidden`).
 
 #### Examples
 
-Common ‘header’ to set up code blocks:
+**Common ‘header’ to set up code blocks:**
 
-````markdown
+````elm
```

Comment:
Should this be `markdown`?

Reply:
We've got an elm code block inside a markdown code block, so the right grammar is ` ````markdown ` + ` ```elm ` here. I'll revert although I understand that some highlighting tools may display the blocks less beautifully that way. Can you share a screenshot? We should probably send a bug report rather than break the semantics in favour of rendering.

Reply:
Oh. It's `markdown-preview-enhanced` that's not highlighting elm syntax in preview if we replace ` ````elm ` with ` ````markdown ` 😅

I thought the issue was in source code highlighting (left side of the screen).